src/main/resources/org/jenkins/plugins/lockableresources/LockStep/help-resource.html

```diff
@@ -0,0 +1,5 @@
+<div>
+	<p>
+		The resource name to lock as defined in Global settings.
```

Comment:
Yes, but it's needed to keep interoperability with freestyle (otherwise the whole design of this plugin would need to be revisited). This case is similar to `tool` step.